src/main/java/weather/chambre/Clouds.java

```diff
@@ -0,0 +1,9 @@
+
+package weather.chambre;
```

Comment:
no idea why you added a `chambre` package and only put some classes inside the package.